pkg/policy/defaultRules.yaml

```diff
@@ -1170,3 +1170,347 @@ rules:
                       required:
                         - retryPolicy
                         - expression
+  - id: 45
+    name: Ensure each container has a read-only root filesystem
+    uniqueName: CONTAINERS_INCORRECT_READONLYROOTFILESYSTEM_VALUE
+    enabledByDefault: false
+    documentationUrl: "https://hub.datree.io/"
```

Comment:
missing documentationUrl

Reply:
melech

Reply:
It's good to add to defaultRules_test.go!

Reply:
@TzlilSwimmer123 added it now